CMakeLists.txt

```diff
@@ -56,10 +56,8 @@ include("dependencies.cmake")
 include_directories(${MACRO_UTILS_INC_FOLDER})
 
 # Include the common build rules for C shared utility
-if(${use_installed_dependencies})
+if(NOT ${use_installed_dependencies})
```

Comment:
>NOT [](start = 3, length = 3)

Was this a bug? Or is this logic flipped from previous version due to specific reasons?

Reply:
This was a bug. Pasting here the conversation with @YoDaMa :

correct. It then means the code of those dependencies should not be expected to be where our submodules are set to be (in each project)

this was expecting the the code for the submodule was there, although the project is being built using installed dependencies

which is incorrect.